### PR TITLE
[Draft] [onert/nnfw_api] Specify loss per epoch

### PR DESCRIPTION
**Pull request by jyoungyun**
ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

/cc @ragmani

### CODE REVIEW

File: tests/nnfw_api/lib/GenModelTrain.h [hunk 1]
```diff
@@ -427,14 +426,17 @@ class GenModelTrain : public ::testing::Test
...
-        {
-          const float actual = actual_losses[i];
-          const float expected = ref_losses[i];
-          EXPECT_NEAR(expected, actual, 0.001) << "Loss #" << i;
+          // TODO better way for handling FP error?
+          for (uint32_t i = 0; i < actual_losses.size(); i++)
+          {
+            const float actual = actual_losses[i];
+            const auto &expected_loss = ref_losses[epoch];
+            float expected =
```
Comment (ragmani) [inline tests/nnfw_api/lib/GenModelTrain.h:435]: (Optional)
```suggestion
            const float expected =
```

**Review by ragmani — COMMENTED** [review]
I think it would lead to much better verification.

**Comment by jyoungyun** [comment]
@nnfw-bot test onert-cross-debug

**Comment by ragmani** [comment]
Error message
```
[2024-06-24T05:15:21.240Z] [ RUN      ] GenModelTrain.BranchOps_FC_Add

[2024-06-24T05:15:21.240Z] nnfw_api_gtest: /opt/jenkins_agent/workspace/nnfw/master/pr-onert-cross-debug/tests/nnfw_api/lib/GenModelTrain.h:178: TrainCaseData uniformTCD(const std::vector<std::vector<std::vector<T> > >&, const std::vector<std::vector<std::vector<T> > >&, const std::vector<std::vector<float> >&) [with T = float]: Assertion `expects_dataset.size() == losses.size()' failed.

[2024-06-24T05:15:21.240Z] /jenkins/aws/workspace/nnfw/master/pr-onert-cross-debug/Product/out/test/command/unittest: line 78:   517 Aborted                 (core dumped) $TEST_BIN $(get_gtest_option)

[2024-06-24T05:15:21.240Z] Product/out/unittest/nnfw_api_gtest failed... return code: 134
```

**Comment by jyoungyun** [comment]
Done!
